lib/galaxy/authnz/psa_authnz.py

```diff
@@ -51,6 +53,8 @@
     "google": "google-openidconnect",
     "globus": "globus",
     "elixir": "elixir",
+    "lifescience": "life_science",
+    "einfracz": "e-infra_cz",
```

Comment:
```suggestion
    "einfracz": "e-infra_cz",
    "nfdi": "nfdi",
```

Reply:
so in my understanding this is mapping to the actual property in a class (e.g. for google [here](https://github.com/python-social-auth/social-core/blob/345ad5ffd786ae11c430dfee2d8b1f58df37be49/social_core/backends/google_openidconnect.py#L11)) However for nfdi the situation looks different because it does some endpoint mapping to subclasses (see [here](https://github.com/python-social-auth/social-core/blob/345ad5ffd786ae11c430dfee2d8b1f58df37be49/social_core/backends/nfdi.py#L54)

Did you test this?